buildspec.saas.yml

```diff
@@ -22,6 +23,25 @@ phases:
     commands:
       - docker push rudderlabs/rudder-transformer:$VERSION
       - docker push rudderlabs/rudder-transformer:latest
+  change_version:
+    commands:
+      - wget https://github.com/mikefarah/yq/releases/download/v4.9.6/yq_linux_amd64 -O /usr/bin/yq
+      - chmod +x /usr/bin/yq
+      - apt update
+      - apt install -y git 
+      - apt update
+      - apt install -y hub
+      - export GITHUB_TOKEN="$GITHUB_TOKEN"
+      - yes | git clone https://$GITHUB_TOKEN@github.com/rudderlabs/rudder-devops.git
+      - cd rudder-devops/helm-charts/shared-services/per-az
+      - git checkout -b version-change-\"$VERSION\"
+      - yq eval ".rudder-transformer.image.tag=\"$VERSION\"" values.enterprise.yaml >> values.enterprise1.yaml
```

Comment:
can this file be updated in place without saving the updated file content in new one ?

Reply:
I've tried the same but not possible since it's not updating the file with yq version:4